build.sbt

```diff
@@ -992,13 +994,28 @@ lazy val launcher = project
   .in(file("engine/launcher"))
   .configs(Test)
   .settings(
+    resolvers += Resolver.bintrayRepo("gn0s1s", "releases"),
     libraryDependencies ++= Seq(
-        "org.scalatest" %% "scalatest" % scalatestVersion % Test,
-        "org.typelevel" %% "cats-core" % catsVersion
+        "org.scalatest"            %% "scalatest"        % scalatestVersion % Test,
+        "org.typelevel"            %% "cats-core"        % catsVersion,
+        "nl.gn0s1s"                %% "bump"             % bumpVersion,
+        "org.apache.commons"        % "commons-compress" % commonsCompressVersion,
+        "org.apache.httpcomponents" % "httpclient"       % apacheHttpClientVersion
       )
   )
   .settings(
-    buildNativeImage := NativeImage.buildNativeImage(staticOnLinux = true).value
+    buildNativeImage := NativeImage
+        .buildNativeImage(
+          "enso",
+          staticOnLinux = true,
+          Seq(
+            "--enable-all-security-services",
```

Comment:
What does this option do?

Reply:
While we use Apache HTTP for the HTTPS connectivity, it still needs some support from Java's stdlib for handling SSL and this option ensures the built image contains these utilities (by default they are not included).